open-api/rest-catalog-open-api.yaml

```diff
@@ -541,6 +541,216 @@ paths:
+                TableDoesNotExist:
+                  $ref: '#/components/examples/NoSuchTableError'
+                NamespaceDoesNotExist:
+                  $ref: '#/components/examples/NoSuchNamespaceError'
+        419:
+          $ref: '#/components/responses/AuthenticationTimeoutResponse'
+        503:
+          $ref: '#/components/responses/ServiceUnavailableResponse'
+        5XX:
+          $ref: '#/components/responses/ServerErrorResponse'
+
+    delete:
+      tags:
+        - Catalog API
+      summary: Uses a `plan-id` as input to cancel a `planTable` operation.
+      operationId: CancelPlan
+      description:
+        Uses a `plan-id` as input to cancel a `planTable` operation.
+        If successful, should return a state that says "cancelled".
+        
+        If an invalid `plan-id` is provided, the service will return a `404 NoSuchPlanIdError` exception indicating that the requested plan does not exist.
+      responses:
+        200:
+          $ref: '#/components/responses/CancelPlanResponse'
+        400:
+          $ref: '#/components/responses/BadRequestErrorResponse'
+        401:
+          $ref: '#/components/responses/UnauthorizedResponse'
+        403:
+          $ref: '#/components/responses/ForbiddenResponse'
+        404:
+          description:
+            Not Found
+            - NoSuchPlanIdException, the plan-id does not exist
+            - NoSuchTableException, the table does not exist
+            - NoSuchNamespaceException, the namespace does not exist
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/IcebergErrorResponse'
+              examples:
+                PlanIdDoesNotExist:
+                  $ref: '#/components/examples/NoSuchPlanIdError'
+                TableDoesNotExist:
+                  $ref: '#/components/examples/NoSuchTableError'
+                NamespaceDoesNotExist:
+                  $ref: '#/components/examples/NoSuchNamespaceError'
+        419:
+          $ref: '#/components/responses/AuthenticationTimeoutResponse'
+        503:
+          $ref: '#/components/responses/ServiceUnavailableResponse'
+        5XX:
+          $ref: '#/components/responses/ServerErrorResponse'
+
+
+  /v1/{prefix}/namespaces/{namespace}/tables/{table}/tasks:
+    parameters:
+      - $ref: '#/components/parameters/prefix'
+      - $ref: '#/components/parameters/namespace'
+      - $ref: '#/components/parameters/table'
+
+    post:
+      tags:
+        - Catalog API
+      summary: Retrieves additional `PlanTask`s and `FileScanTask`s from the service.
+      operationId: RetrieveTasks
+      description:
+        Requires a client to provide a `PlanTask` which can be obtained from `GetTaskStatus`, 
+        in order to fetch additional `PlanTask`s and `FileScanTask`s from a service.
+      
+        If an invalid plan-task is provided to the service, the service will return a `404 NoSuchPlanTaskError` exception.
```

Comment:
This doesn't need to be in the description. It is already covered by the response codes below. No need to duplicate.